agent/api.mdx

```diff
@@ -225,32 +225,194 @@ GET/api/tunnels/:name
 
 ## Stop tunnel
 
-Stop a running tunnel
+Stop a running tunnel.
```

Comment:
We gotta circle back on this page at some point and link it out to a doc that explains what a tunnel is. I don't know how a tunnel differs from an ingress, or whether a tunnel is an ngrokism or an industry-standard technical term.

Linear issue to track it
- https://linear.app/ngrok/issue/DOC-431/improve-docs-on-tunnels